### ISSUE TITLE
Import documentation 

### ISSUE THREAD

**Issue by boyravikumar** [nmalex/threejs.dlx#1]
Hi, 

How can add this plugin to 3ds max. I am a very begineer in 3ds max. I have downloaded the zip, extracted it. Then added extracted folder  to 3rd party plugins from customize menu -> Configure plugin paths. 

Affter doing this how can export the model in 3ds max to three.js json. 

Please help.

**Comment by nmalex** [comment]
Hi,
the plugin is not exactly what you need. As I understood, you need to export the model in 3ds max to three.js json. But this project does exactly opposite - imports three.js json into 3dsmax.

**Comment by boyravikumar** [comment]
Ok, thats perefect, which is what i need. I need my three js scene to open in 3ds max, adjust the light and get good rendered image with v ray. Renderfarm.js looks like its a complete automation of this process. This automation is as well required for our project but in the mean time we want to try out manually importing three js scene / objects in to 3ds max (along with materials including the image textures).

**Comment by nmalex** [comment]
That's right, you can import json in max. 
See, the source code here you will need to compile into DLL and then rename it to *.dlx and add to plugins folder. After 3dsmax restart it will add new command to MaxScript which you need to call to actually import mesh. 
It is important that you understand that it imports only 1 mesh at once. It won't import scene with node hierarchy, transformations, rotations etc. Only 1 mesh.

Secondly, what maybe important is 3dsmax version. The source here is developed for 3dsmax SDK version 2018. In case you have different version of 3dsmax, it maybe required.to do some adjustments.
Please let me know if it makes sense.

**Comment by boyravikumar** [comment]
Hi, finally i am able to add the plugin. threejsImportBufferGeometry commad was added and able to import the objects which were in console/testdata. Also, i went through the parsing logic (threejs.core.cpp) , looks whats here is very basic one to import the buffer geometry, Also, there were no materials (color & image textures) are parsed. Do i have to extend this logic to import along with the materials.

**Comment by nmalex** [comment]
Hi,
good news. That's right, it only imports single piece of BufferGeometry and that's it. I did not aim to import the whole three.js serialized scene (as it was not needed for main project), but logically these feature belong here too. Of course I don't mind if you decide to improve this sub-project. I will support it and be ready to do code review and merge branches for you.

**Comment by boyravikumar** [comment]
I might have to do this finally. But have to think of time it takes and is it worthy to invest on this. Also, before starting this  
1. I want to confirm  if there are any alternatives (any open source programs) which does this. 
2. Exporting to obj from threejs and importing it in 3ds max works fine except the meshes with image textures. Should we fix this from three.js side
3. Also, is it possible to write the python code for max rather than cop. My intent is can we add python plugins to 3ds max.
4. Also, there is gltf exporter in three js, does max support importing gltf, if not now atleast in next releases of 3ds max

Please share your inputs on this.

**Comment by boyravikumar** [comment]
Hi, 

How did you get this furniture demo. https://www.youtube.com/watch?v=uznzlDO8frU&feature=youtu.be&t=35

**Comment by nmalex** [comment]
Hi, this demo is website (you can see code in browser) 
plus https://github.com/nmalex/renderfarm.js-client
plus https://github.com/nmalex/renderfarm.js-server running on the backend together with threejs.dlx and other dlx plugins available here. Deployment is tricky and very undocumented. I did not do that due to so low interest to the project.
